src/streams/StreamIcon.js

```diff
@@ -8,12 +8,20 @@ type Props = $ReadOnly<{|
   color?: string,
   isPrivate: boolean,
   isMuted: boolean,
+  isSubscribed: boolean,
   size: number,
   style?: TextStyleProp,
 |}>;
 
-export default ({ color, style, isPrivate = false, isMuted = false, size }: Props) => {
-  const StreamIcon = isMuted ? IconMute : isPrivate ? IconPrivate : IconStream;
+export default ({
+  color,
+  style,
+  isPrivate = false,
+  isMuted = false,
+  isSubscribed = true,
+  size,
+}: Props) => {
+  const StreamIcon = isMuted && !isMuted ? IconMute : isPrivate ? IconPrivate : IconStream;
```

Comment:
I think you meant to use `isSubscribed` in here somewhere.

Reply:
Good catch! Fixed.